bin/pycbc_live

```diff
@@ -260,12 +260,20 @@ class LiveEventManager(object):
             f.attrs['command_line'] = sys.argv
             f.attrs['num_live_detectors'] = len(self.live_detectors)
 
+            def h5py_unicode_workaround(stuff):
+                # FIXME workaround for the fact that h5py cannot handle Unicode
```

Comment:
Why is this a FIXME? h5py is a binary format, it's not going to be able to handle non-binary strings. python3 is just being clear that this encoding is needed for strings.

Reply:
I guess one potential approach is to encode the string when defining the array in the first place:
```
>>> a = np.array(u'asdasd', dtype='S')
>>> a
array(b'asdasd', 
      dtype='|S6')
```

Reply:
Yeah, apparently [h5py will never handle the U type transparently](http://docs.h5py.org/en/stable/strings.html#what-about-numpy-s-u-type), so I removed the FIXME.

Reply:
Your alternative approach of always using arrays of ASCII strings might be a better choice. I will see if it involves a more complicated change.

Reply:
I tried the alternative solution by explicitly converting the approximant array to dtype `S` in the bank code, but that is not enough; more changes are needed to allow carrying around byte buffers instead of strings. I think I will keep the original solution.